web/org.eclipse.xtend.web/src/main/xtend-gen/org/eclipse/xtend/web/ServerLauncher.java

```diff
@@ -0,0 +1,88 @@
+/**
```

Comment:
I'm not sure what the Xtext project policy re. git commiting generated source code in  xtend-gen folders (in the Xtext project's own source tree), but personally I would NOT include any of these generated files in the PR, and instead just use the http://xtext.github.io/xtext-gradle-plugin/xtend.html in the build.gradle.